packages/synapse-react-client/src/components/widgets/query-filter/FacetFilterControls.integration.test.tsx

```diff
@@ -145,41 +106,6 @@ describe('FacetFilterControls tests', () => {
   })
 
   describe('handling child component callbacks', () => {
-    it('should propagate range correctly', async () => {
```

Comment:
This logic was pulled out of this component, so it doesn't need to be tested here.